config/sqlserver/sample_tpch_config.xml

```diff
@@ -10,6 +10,8 @@
     <reconnectOnConnectionFailure>true</reconnectOnConnectionFailure>
     <isolation>TRANSACTION_SERIALIZABLE</isolation>
     <batchsize>1024</batchsize>
+    <!-- Session setup statements file -->
+    <!-- <sessionsetupfile>config/sqlserver/session_setup_sqlserver_cmds_example.sql</sessionsetupfile> -->
```

Comment:
Request that we have at least one example config where this option is not commented out so that it runs during the CI tests and/pr add an explicit unit test for it.

Reply:
@ranaalotaibiMS bump :)